### DISCUSSION TITLE
Getting the axon ID numbers showing up on segmented image

### DISCUSSION THREAD

**Discussion by Paasu** [neuropoly/axondeepseg#550] [Help needed] [answered]
Hello all,

I want to know how to get the axon ID numbers showing up on the segmented image. It already happens on FSLeyes but FSLeyes isn't working on my linux or windows versions of ADS and I do not own a mac. The morphmetrics file spits out the axon ID in the first column but I do not have anything to match it up to in the images so I don't know which measurement corresponds to which axon which is something I would like to know.

Thanks

**Comment by mathieuboudreau** [comment, marked as answer]
Hi @Paasu,

Thanks for your question. The command-line tool `axondeepseg_morphometrics` is what you're looking for I believe. If you call that with the `-i IMAGE` (where image is the path to your axon-myelin mask) and `-s PIXEL_SIZE` (where PIXEL_SIZE is the size of your pixel in micrometers), then I believe this should give you the file(s) you want.

**Reply by Paasu** [reply]
Ah right I already have the morphometrics file using CLI, the problem is that the first column in that file has the axon number/ID but it doesn't correspond to any number in the segmented image because there are no numbers on it. FSLeyes shows the numbers but FSLeyes doesn't work for me at the moment.

**Reply by mathieuboudreau** [reply]
Ahh, the `axondeepseg_morphometrics` doesn't output a file with the axon IDs? @Stoyan-I-A ?

**Reply by mathieuboudreau** [reply]
If that's the case @Paasu, we'll open an issue to resolve that.

**Reply by Paasu** [reply]
The `axondeepseg_morphometrics` does output it with the IDs, the image doesn't have any IDs however which is what I'm interested in.

**Reply by mathieuboudreau** [reply]
Sorry, I meant an image with the IDs. I believe we added this recently, but thought it was default. The suffix of the image should be `_axonmyelin_index.png` if it's exported correctly with that CLI command.

**Reply by Paasu** [reply]
I updated my ADS with the new changes and it the index file is also generated. My bad!
Thank you very much for helping me out.

**Reply by mathieuboudreau** [reply]
Ah ha! Great! Thanks for reaching out – I should have checked first if you had the latest version. Cheers.
